packages/webapp/.env

```
@@ -1,5 +1,7 @@
-NEXT_PUBLIC_API_URL=http://api.local.com
-#NEXT_PUBLIC_API_URL=https://api.daily.dev
+#NEXT_PUBLIC_API_URL=http://api.local.com
+NEXT_PUBLIC_API_URL=https://api.daily.dev
+
+
```

Just reverting the env variables 🚀 

```suggestion
NEXT_PUBLIC_API_URL=http://api.local.com
#NEXT_PUBLIC_API_URL=https://api.daily.dev
```